### ISSUE TITLE
"appid" does not find any games

### ISSUE THREAD

**Issue by L1Q** [nning/protonutils#6]
```
[l1q@l1q-mj ~]$ protonutils list -u 82996121
Proton 5.13-6
        Everyday Shooter
        FlatOut 2
        Micro Platformer
        Quake III Arena
        Quake III: Team Arena

Proton 6.3-8
        Assassin's Creed
        Grand Theft Auto III
        Grand Theft Auto: Vice City
        The Binding of Isaac: Rebirth

Proton 6.3-8 (Default)
        Nuclear Throne
        Remnants of Naezith
        Sky Force Reloaded

Proton-6.12-GE-1
        A Glider's Journey

Proton-6.14-GE-2
        The Witness

Proton-6.19-GE-1
        Assassin's Creed II
        Muck

Proton-6.21-GE-1
        The Typing of The Dead: Overkill

[l1q@l1q-mj ~]$ protonutils appid "The Typing of The Dead: Overkill"
Warning: Several Steam users available, using 69221152
All available users: 69221152, 82996121
Option "-u" can be used to specify user

[l1q@l1q-mj ~]$ protonutils appid "The Witness"
Warning: Several Steam users available, using 69221152
All available users: 69221152, 82996121
Option "-u" can be used to specify user

[l1q@l1q-mj ~]$ protonutils appid "Muck"
Warning: Several Steam users available, using 69221152
All available users: 69221152, 82996121
Option "-u" can be used to specify user

[l1q@l1q-mj ~]$
```

**Comment by nning** [comment]
Thanks for reporting this issue!

It seems I broke this feature with one of my yesterday's commits by renaming the app ID cache: 9c815dc
Currently the automatic test coverage is bad, because I'm not sure how to include test fixtures (`appinfo.vdf`, for example) into the repository. And in my manual tests, the old cache file was still existing.

I fixed it in 90302c8 and created a new release:
https://github.com/nning/protonutils/releases/tag/v1.2.8
